x-pack/plugins/ml/public/application/contexts/ml/ml_context.ts

```diff
@@ -12,17 +12,17 @@ import { MlServicesContext } from '../../app';
 
 export interface MlContextValue {
   combinedQuery: any;
-  currentIndexPattern: DataView; // TODO this should be IndexPattern or null
+  currentDataView: DataView; // TODO this should be IndexPattern or null
   currentSavedSearch: SavedSearchSavedObject | null;
-  indexPatterns: DataViewsContract;
+  dataViewsContract: DataViewsContract;
   kibanaConfig: any; // IUiSettingsClient;
   kibanaVersion: string;
 }
 
 export type SavedSearchQuery = object;
 
 // This context provides dependencies which can be injected
-// via angularjs only (like services, currentIndexPattern etc.).
+// via angularjs only (like services, currentDataView etc.).
```

Comment:
Seeing as you are editing this comment, probably a good time to drop the old reference to AngularJS. `angularjs` is also used a few lines below.

Reply:
Updated in https://github.com/elastic/kibana/pull/116455/commits/7ee9241872df9cecf890ae7dce916bfa3f491d84